tests/test_fields.py

```diff
@@ -93,6 +93,13 @@ def test_number_fields_prohbits_boolean(self):
             strict_field.serialize('value', {'value': True})
         assert excinfo.value.args[0] == 'Not a valid number.'
 
+    def test_number_field_overflow(self):
+        strict_field = fields.Float()
+        with pytest.raises(ValidationError) as excinfo:
+            strict_field.serialize('value', {'value': 999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999990})
```

Comment:
You could probably simplify this constant to `2**1024`. It would be nice if it was based on `sys.float_info.max`, but simply adding to that value doesn't seem to cause an overflow.